kubernetes/manifest-templates/velum.yaml

```diff
@@ -82,3 +90,6 @@ spec:
     - name: velum-source-code
       hostPath:
         path: ${project_dir}
+    - name: velum-secrets
+      hostPath:
+        path: /tmp/velum-secrets
```

Comment:
nevertheless, as it is rails app specific, can't we always use the rails tmp dir here? then we wouldn't need an environment variable nor a volume mount.